openedx/core/djangoapps/content_libraries/tests/test_views_lti.py

```diff
@@ -0,0 +1,55 @@
+"""
+Tests for LTI views.
+"""
+
+from django.test import TestCase
+from django.urls import reverse
+
+from openedx.core.djangoapps.content_libraries.constants import PROBLEM
+from openedx.core.djangoapps.content_libraries.views import LibraryBlockLtiUrlView
+
+from .base import URL_LIB_LTI_JWKS
+from .base import skip_unless_cms, ContentLibrariesRestApiTest
+
+
+
```

Comment:
Too many blank lines before the class definition is making pylint unhappy.

Reply:
Oh, I accidentally hit some newlines. I'll fix that today/tomorrow!